assets/js/components/Link.js

```diff
@@ -27,143 +27,154 @@ import { Link as RouterLink } from 'react-router-dom';
  * WordPress dependencies
  */
 import { _x } from '@wordpress/i18n';
+import { forwardRef } from '@wordpress/element';
 
 const BUTTON = 'BUTTON';
 const BUTTON_DISABLED = 'BUTTON_DISABLED';
 const EXTERNAL_LINK = 'EXTERNAL_LINK';
 const LINK = 'LINK';
 const ROUTER_LINK = 'ROUTER_LINK';
 
-function Link( {
-	'aria-label': ariaLabelProp,
-	arrow,
-	back,
-	caps,
-	children,
-	className,
-	danger,
-	disabled,
-	external,
-	hideExternalIndicator,
-	href,
-	inherit,
-	inverse,
-	onClick,
-	small,
-	to,
-	...otherProps
-} ) {
-	const getType = () => {
-		// Force button element if `onClick` prop is passed and there's no `href`
-		// or `to` prop.
-		if ( ! href && ! to && onClick ) {
-			if ( disabled ) {
-				return BUTTON_DISABLED;
+const Link = forwardRef(
```

Comment:
This was needed because the `Tab` components attach a `ref` to the components they render.

In general this is a nice-to-have and doesn't affect the component in any way besides offering refs to parent components that need it.